### PR TITLE
Fix license and add keywords

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
License should be a valid [SPDX license expression](https://spdx.org/licenses/).

Keywords are used in the [new module list](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/) I'm working on.

### CODE REVIEW

**Comment by KristjanESPERANTO** [comment]
@x3mEr Are you still maintaining this module? 🙂

File: package.json [hunk 1]
```diff
@@ -1,11 +1,15 @@
 {
   "name": "MMM-MP3Player",
-  "description": "A MagicMirror module for playing music from folder",
+  "description": "A MagicMirror² module for playing music from folder.",
```
Comment (x3mEr) [inline package.json:3]: Contains a special symbol. Please, remove "²".

**Review by x3mEr — CHANGES_REQUESTED** [review]

**Comment by x3mEr** [comment]
Thank you for your commit!) All good, except this special symbol "²". Don't think it's a good idea to include it into the json file.

**Comment by KristjanESPERANTO** [comment]
I wonder why you call it special. The ² is not a special character at all. It is simply a squared two and is part of the official spelling of MagicMirror². A normal mathematical character and in ISO-8859-1 since 1998 and also in UTF-8. For the description in package.json it is normal to use UTF-8 characters.

Are you sure I should remove it?

**Comment by x3mEr** [comment]
It's not in Unicode.

**Comment by KristjanESPERANTO** [comment]
It's Unicode U+00B2: http://www.unicode-symbol.com/u/00B2.html

**Comment by x3mEr** [comment]
Oops, was thinking about "ASCII" but has wrote "Unicode".
Nevertheless, you are right: "in json it is normal to use UTF-8 characters".

**Comment by KristjanESPERANTO** [comment]
No problem 🙂 Fortunately, there are now fewer and fewer systems that are limited to ASCII. Thanks for merging!

**Comment by KristjanESPERANTO** [comment]
Your module now looks nice on [the new module list](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/) :smiley:

**Pull request merged**